src/SpotifyApi.NetCore/FollowApi.cs

```diff
@@ -204,7 +219,7 @@ public FollowApi(HttpClient httpClient) : base(httpClient)
                     ArgumentNullException("playlistId");
 
             var builder = new UriBuilder($"{BaseUrl}/playlists/{playlistId}/followers");
-            await Put(builder.Uri, isPublic, accessToken);
+            await Put(builder.Uri, new { @public = isPublic }, accessToken);
```

Comment:
This looks like a breaking change to a Released API - have you got a test that asserts that this still works?

Reply:
Hi Daniel,

Thank you for taking time out for this.  Will triple check.

Thanks for your patience and time.

Best Regards,

Akshay Srinivasan

Reply:
Hi Daniel,

My logic was that the SpotifyAPI documentation expects the value's property name to be the Json string "public".  While somehow it worked it is not perfectly correct the original code. This is why I changed it.  Working on unit tests to verify this works perfectly well.

Thanks for your patience and time.

Best Regards,

Akshay Srinivasan

Reply:
Hi Daniel,

One scenario of why this would have worked originally is that the parameter value on Spotify's end is accessing via index and hence wrong parameter name as per the documentation works anyways on this API call.

Thanks for your patience and time.

Best Regards,

Akshay Srinivasan

Reply:
Hi Daniel,

My suspicion was right.  The parameter name can be anything and does not have to be called "public" as per the SpotifyAPI documentation as on their end they are not using the parameter name to get to the value.  Technically this code change is not required at all.

How do I know this I changed @public = ... to @xyz = ... and it passed with no errors.

So, for the time being, I will revert this change as this works always until they change the code in their API to not use the index but use the parameter name as per their documentation to access the value.

So as of this message I am done from my side. The ball is back in your court Daniel ♥♥♥.

Thanks for your patience and time.

Best Regards,

Akshay Srinivasan